### ISSUE TITLE
The negative log likelihood Not update over epochs

### ISSUE THREAD

**Issue by iris1104** [jaredleekatzman/DeepSurv#13]
I generated nonlinear survival data using the SimulatedData class in your package, and only got 0.54 c-index for the test dataset. The negative log likelihood plot showed two separate straight lines for the training data and validation data. I think the loss function did not converge over epochs during my training process. I attached the code I used below. Could you please help me find the reason? Thank you very much!!

def generate_data(treatment_group = False):
    np.random.seed(123)
    sd = SimulatedData(5, num_features = 9,
        treatment_group = treatment_group)
    train_data = sd.generate_data(5000)
    valid_data = sd.generate_data(2000)
    test_data = sd.generate_data(2000)

    return train_data, valid_data, test_data

train_data,valid_data,test_data=generate_data(True)
hyperparams = {
'n_in': 10,
'learning_rate': 1e-5,
'hidden_layers_sizes':[10,10]
}

network = DeepSurv(**hyperparams)
log =network.train(train_data, valid_data, n_epochs=500)
network.get_concordance_index(**test_data)
deepsurv.plot_log(log)

**Comment by scienceaiden** [comment]
i am surprised that it even ran. you num_features 9 but 'n_in': 10 - they should be equal
